packages/flutter/test/rendering/layers_test.dart

```diff
@@ -109,4 +111,132 @@ void main() {
     expect(leaderLayer.debugSubtreeNeedsAddToScene, true);
     expect(followerLayer.debugSubtreeNeedsAddToScene, true);
   });
+
+  void checkNeedsAddToScene(Layer layer, void mutate()) {
+    layer.debugMarkClean();
+    layer.updateSubtreeNeedsAddToScene();  // ignore: invalid_use_of_protected_member
+    expect(layer.debugSubtreeNeedsAddToScene, false);
+    mutate();
```

Comment:
nit: rename this argument "mutateCallback" or something to make it clear that this is an argument and not a method... I just spent a few minutes searching for the "mutate()" method. :-)

Reply:
Done.